pkgs/development/python-modules/pkg-about/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, importlib-metadata
+, importlib-resources
+, setuptools
+, packaging
+, tomli
+}:
+
+buildPythonPackage rec {
+  pname = "pkg-about";
+  version = "1.0.8";
+  disabled = pythonOlder "3.7";
+
+  format = "pyproject";
+  src = fetchPypi {
+    pname = "pkg_about";
+    inherit version;
+    extension = "zip";
+    sha256 = "mb43XbKypgilagXLW33kP8wXxioNsfLtl6AEnOI1WlA=";
+  };
+
+  nativeBuildInputs = [
+    packaging
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    importlib-metadata
+    importlib-resources
+    packaging
+    setuptools
+    tomli
+  ];
+
```

Comment:
The upstream repo contains [tests](https://github.com/karpierz/pkg_about/tree/1.0.8/tests). Tests should be executed if possible otherwise add `doCheck = false;` with a comment.

Please add `pythonImportsCheck` anyways.

Reply:
I thought tests were already run automatically, is this not the case?

Reply:
They are but here we get:

```bash
[...]
Finished executing pythonRemoveTestsDir
running install tests
Running tests


----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```